test/parallel/test-https-agent-getname.js

```diff
@@ -12,7 +12,7 @@ const agent = new https.Agent();
 // empty options
 assert.strictEqual(
   agent.getName({}),
-  'localhost:::::::::::::::::'
+  'localhost:::::::::::::::::::'
```

Comment:
Doesn't this make this semver-major?

Reply:
I don't know. The docs claim its an opaque string. getName() is mostly intended for internal use as a hash key, I think. If it has to be semver-major, that's OK, openssl 1.1.1 is itself semver-major. Still ,this is a nice API, it would be good to get out on current.

Reply:
yeah, the docs just say it includes bits of tls info, not what bits. But lack of docs doesn't make it not an API (unfortunately). I just can't think of a use for it that would involve people parsing the agent name to find the bits, and then to complain to us when the bits had changed. Lets let some other folks weigh in. Maybe I'll start off calling semver-major, we can downgrade if that seems excessive.

Reply:
It's probably fine. We've changed it before mid-release cycle and no one ever complained, AFAIK.

Reply:
@refack ok to resolve this, then, and leave as semver-minor?

Reply:
Done